core/src/main/java/org/apache/gravitino/storage/relational/mapper/OwnerMetaMapper.java

```diff
@@ -0,0 +1,161 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.storage.relational.mapper;
+
+import org.apache.gravitino.storage.relational.po.OwnerRelPO;
+import org.apache.ibatis.annotations.Delete;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Param;
+import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
+
+/**
+ * A MyBatis Mapper for owner meta operation SQLs.
+ *
+ * <p>This interface class is a specification defined by MyBatis. It requires this interface class
+ * to identify the corresponding SQLs for execution. We can write SQLs in an additional XML file, or
+ * write SQLs with annotations in this interface Mapper. See: <a
+ * href="https://mybatis.org/mybatis-3/getting-started.html"></a>
+ */
+public interface OwnerMetaMapper {
+
+  String OWNER_TABLE_NAME = "owner_meta";
+
+  @Select(
+      "SELECT metalake_id as metalakeId,"
+          + " owner_id as ownerId,"
+          + " owner_type as ownerType,"
+          + " metadata_object_id as metadataObjectId,"
+          + " metadata_object_type as metadataObjectType,"
+          + " audit_info as auditInfo,"
+          + " current_version as currentVersion, last_version as lastVersion,"
+          + " deleted_at as deletedAt"
+          + " FROM "
+          + OWNER_TABLE_NAME
+          + " WHERE metadata_object_id = #{metadataObjectId} AND"
+          + " metadata_object_type = #{metadataObjectType}"
+          + " AND deleted_at = 0")
+  OwnerRelPO selectOwnerMetaByMetadataObjectIdAndType(
+      @Param("metadataObjectId") Long metadataObjectId,
+      @Param("metadataObjectType") String metadataObjectType);
+
+  @Insert(
+      "INSERT INTO "
+          + OWNER_TABLE_NAME
+          + "(metalake_id, metadata_object_id, metadata_object_type, owner_id, owner_type,"
+          + " audit_info, current_version, last_version, deleted_at)"
+          + " VALUES ("
+          + " #{ownerRelPO.metalakeId},"
+          + " #{ownerRelPO.metadataObjectId},"
+          + " #{ownerRelPO.metadataObjectType},"
+          + " #{ownerRelPO.ownerId},"
+          + " #{ownerRelPO.ownerType},"
+          + " #{ownerRelPO.auditInfo},"
+          + " #{ownerRelPO.currentVersion},"
+          + " #{ownerRelPO.lastVersion},"
+          + " #{ownerRelPO.deletedAt}"
+          + ")")
+  void insertOwnerRel(@Param("ownerRelPO") OwnerRelPO ownerRelPO);
+
+  @Update(
+      "UPDATE "
+          + OWNER_TABLE_NAME
+          + " SET deleted_at = (UNIX_TIMESTAMP() * 1000.0)"
+          + " + EXTRACT(MICROSECOND FROM CURRENT_TIMESTAMP(3)) / 1000"
+          + " WHERE metadata_object_id = #{metadataObjectId} AND metadata_object_type = #{metadataObjectType} AND deleted_at = 0")
+  void softDeleteOwnerRelByMetadataObjectIdAndType(
+      @Param("metadataObjectId") Long metadataObjectId,
+      @Param("metadataObjectType") String metadataObjectType);
```

Comment:
Can we use `MetadataObject.type` type instead `String` for `metadataObjectType` param?
Beacuse if we can use `MetadataObject.type` type, the type parameters are very clear, 
and we can only call once `MetadataObject.type.name()` function, currently we need to call `.name()` function repeatedly.

Reply:
`TagMetadataObjectRelMapper` uses String instead of Enum type. 
Our PO uses String instead of Enum type. So we would better use String here.